notebooks/image_models/solutions/classification_automl_vision.ipynb

```diff
@@ -0,0 +1,675 @@
+{
```

Comment:
<p>I think it's important we have a reference for how to do this all programmatically with the Vertex API like this. Since this whole lab is using the UI perhaps we could keep it and append a _ui in the lab name and also create a lab that does the same thing with the api (below) and let people choose which they prefer to walk through. </p><p> </p><pre class="ql-syntax" spellcheck="false">ds = aiplatform.ImageDataset.create(
&nbsp; &nbsp; &nbsp; &nbsp; display_name=display_name,
&nbsp; &nbsp; &nbsp; &nbsp; gcs_source=src_uris,
&nbsp; &nbsp; &nbsp; &nbsp; import_schema_uri=aiplatform.schema.dataset.ioformat.image.single_label_classification,
&nbsp; &nbsp; &nbsp; &nbsp; sync=sync,
&nbsp; &nbsp; )
job = aiplatform.AutoMLImageTrainingJob(
&nbsp; &nbsp; &nbsp; &nbsp; display_name=display_name,
&nbsp; &nbsp; &nbsp; &nbsp; model_type=model_type,
&nbsp; &nbsp; &nbsp; &nbsp; prediction_type="classification",
&nbsp; &nbsp; &nbsp; &nbsp; multi_label=multi_label,
&nbsp; &nbsp; )


&nbsp; &nbsp; model = job.run(
&nbsp; &nbsp; &nbsp; &nbsp; dataset=ds,
&nbsp; &nbsp; &nbsp; &nbsp; model_display_name=model_display_name,
&nbsp; &nbsp; &nbsp; &nbsp; training_fraction_split=training_fraction_split,
&nbsp; &nbsp; &nbsp; &nbsp; validation_fraction_split=validation_fraction_split,
&nbsp; &nbsp; &nbsp; &nbsp; test_fraction_split=test_fraction_split,
&nbsp; &nbsp; &nbsp; &nbsp; budget_milli_node_hours=budget_milli_node_hours,
&nbsp; &nbsp; &nbsp; &nbsp; disable_early_stopping=disable_early_stopping,
&nbsp; &nbsp; &nbsp; &nbsp; sync=sync,
&nbsp; &nbsp; )
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>